org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandlerTest.java

```diff
@@ -1459,7 +1459,7 @@ public void testSnippet_record() throws Exception {
 		assertFalse(items.isEmpty());
 		items.sort((i1, i2) -> (i1.getSortText().compareTo(i2.getSortText())));
 
-		CompletionItem item = items.get(10);
+		CompletionItem item = items.get(11);
```

Comment:
Is there any new factor that contributes more completion items?

Reply:
See https://www.eclipse.org/eclipse/news/4.17/jdt.php#content-assist-substring-types